src/components/application_manager/src/hmi_capabilities_impl.cc

```diff
@@ -339,6 +339,8 @@ void InitCapabilities() {
   image_field_name_enum.insert(
       std::make_pair(std::string("locationImage"),
                      hmi_apis::Common_ImageFieldName::locationImage));
+  image_field_name_enum.insert(std::make_pair(
```

Comment:
@IGapchuk Check coding style please. As I can see your line is different than the others in the file

Reply:
@AByzhynar It's not a coding style formatting issue. Coding style formatting is right. It looks like that , because `"alertIcon"` and `hmi_apis::Common_ImageFieldName::alertIcon` lexemes are shorter that `"locationImage"` and `hmi_apis::Common_ImageFieldName::locationImage`.